### PR TITLE
Include query ids in failed query creates #886  - Depends on base-rest-responses change

### PR DESCRIPTION
**Pull request by jzgithub1**
Added query id to the BaseResponse for queries.  This PR depends on changes in the base-rest-reponses project.

### CODE REVIEW

**Comment by jzgithub1** [comment]
Solves #886

File: pom.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
         <version.microservice.accumulo-utils>1.3</version.microservice.accumulo-utils>
         <version.microservice.audit-api>1.4</version.microservice.audit-api>
         <version.microservice.authorization-api>1.4</version.microservice.authorization-api>
-        <version.microservice.base-rest-responses>1.2</version.microservice.base-rest-responses>
```
Comment (keith-ratcliffe) [inline pom.xml:83]: I might be confused. There's already a 1.2 base-rest-responses jar deployed to github, so I'm not understanding how this PR can be dependent on a 1.2-SNAPSHOT. This PR should depend on a new 1.3 release. Likewise, your current base-rest-responses PR should be targeting 1.3-SNAPSHOT at the moment

Right?

Comment (jzgithub1) [reply pom.xml:83]: @keith-ratcliffe ,  @ivakegg  wanted me to do it this way I am pretty sure.  I had to change some code in that project  (base-rest-responses) for this pull request.  That is to say that there is an associated pull request in that project which I mention above.

Comment (ivakegg) [reply pom.xml:83]: I will fix the version when I merge this in.

**Pull request closed without merging**